app/assets/stylesheets/cases.scss

```diff
@@ -263,13 +263,23 @@ $color-dark-gray: #4D4D4D;
 }
 
 #unfollow-button {
-  // font-size: small;
   color: white;
-  background-color: $color-gray;
+  background-color: $color-orange;
```

Comment:
@rlgreen91 Is this a consolidation of CSS files? I feel like it may be helpful to separate this into another PR in order to keep things manageable.

Reply:
This is part of the situation I outlined in the comment below.